plugins/proxmox/proxmox_count

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#%# family=auto
+#%# capabilities=autoconf
+
+# Michael Grote
+# michael.grote ät posteo.de
+# Outputs the running, stopped and total counts of ProxMox VMs and LXCs.
+# needs to be run as user: root AND group: root
+# [proxmox_count]
+# user root
+# group root
+
+# Pfade zu binaries
+wqm=$(command -v qm)
+wpct=$(command -v pct)
+# Berechne VMs
+total_vm=$($wqm list | sed 1d | wc -l)
```

Comment:
In order to avoid inconsistencies (e.g. a machine was started while the plugin is running), you should retrieve the `$wqm list` just once. Afterwards you can process this output multiple times.

Reply:
good idea, changed it